integrations/spark/openhouse-spark-runtime/src/main/antlr/com/linkedin/openhouse/spark/sql/catalyst/parser/extensions/OpenhouseSqlExtensions.g4

```diff
@@ -83,6 +84,26 @@ columnRetentionPolicy
     : ON columnNameClause (columnRetentionPolicyPatternClause)?
     ;
 
+replicationPolicy
+    : REPLICATION '=' tableReplicationPolicy
+    ;
+
+tableReplicationPolicy
+    : '(' replicationPolicyClause (',' replicationPolicyClause)* ')'
+    ;
+
+replicationPolicyClause
+    : '{' replicationPolicyClusterClause (',' replicationPolicyIntervalClause)? '}'
+    ;
+
+replicationPolicyClusterClause
+    : CLUSTER ':' STRING
+    ;
+
+replicationPolicyIntervalClause
+    : INTERVAL ':' STRING
+    ;
```

Comment:
Can we reuse `duration` instead of string? It adds some restrictions in the SQL layer.

Reply:
this will cause the error displayed for an incorrect input to be `mismatched input ''24h'' expecting {RETENTION_DAY, RETENTION_YEAR, RETENTION_MONTH, RETENTION_HOUR}` which may not be accurate for the input, but I can adjust the SQL to be more restrictive

Reply:
Yes, makes sense. Can limit it to only HOUR duration though.
cc @chenselena